docs/index.yml

```diff
@@ -298,7 +290,7 @@ additionalContent:
       # Card
       - title: ".NET API reference"
         summary: API reference documentation for .NET
-        url: ../api/index.md?view=net-5.0
+        url: ../api/index.md?view=net-6.0
       # Card
       - title: ".NET Core API reference"
```

Comment:
I don't think we should have separate cards for .NET and .NET Core API ref. So I suggest removing the .NET Core API ref card.